### ISSUE TITLE
[Error] TypeError at /reports/download/

### ISSUE THREAD

**Issue by tahersb** [PearsonEducation/bag-of-holding#1]
Received TypeError at /reports/download/ when attempting to download reports at : 
1. http://host:8000/reports/download/?report_type=engagement_coverage
2. http://host:8000/reports/download/?report_type=threadfix_summary

Error: 
<code>
TypeError at /reports/download/
super() takes at least 1 argument (0 given)
Request Method: POST
Request URL:    http://host:8000/reports/download/?report_type=engagement_coverage
Django Version: 1.8.2
Exception Type: TypeError
Exception Value:  
super() takes at least 1 argument (0 given)
Exception Location: \bag-of-holding\project\boh\reports.py in **init**, line 41
Python Executable: \Python27\python.exe
Python Version: 2.7.8
Python Path:  
['\bag-of-holding\project',
 '\Python27\lib\site-packages\setuptools-18.0.1-py2.7.egg',
 '\Python27\lib\site-packages\pip-7.1.0-py2.7.egg',
 '\Windows\system32\python27.zip',
 '\Python27\DLLs',
 '\Python27\lib',
 '\Python27\lib\plat-win',
 '\Python27\lib\lib-tk',
 'C:\Python27',
 'C:\Python27\lib\site-packages']

</code>

Traceback located at : http://dpaste.com/0M92YPP

**Comment by aparsons** [comment]
Thanks for the issue report. I have found the source of the issue and corrected it. 

The source of the problem was Python 2's handling of super classes. It works just fine on Python 3, but needed a little extra code for Python 2. That has been added and it should work now on Python 2.

Thanks so much for creating this issue and using Bag of Holding!

Fixed here: https://github.com/PearsonEducation/bag-of-holding/commit/4fa983bccb925b66ada8b8f712cfb3ebdcb4c830
